cmd/node-joiner/main.go

```diff
@@ -22,30 +22,26 @@ func main() {
 func nodeJoiner() error {
 	nodesAddCmd := &cobra.Command{
 		Use:   "add-nodes",
-		Short: "Generates an ISO that could be used to boot the configured nodes to let them join an existing cluster",
+		Short: "Generates an ISO that can be used to boot the configured nodes to let them join an existing cluster",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			kubeConfig, err := cmd.Flags().GetString("kubeconfig")
+			dir, kubeConfig, err := getDirAndKubeconfigValuesFromFlags(cmd)
 			if err != nil {
 				return err
 			}
-			dir, err := cmd.Flags().GetString("dir")
+			generatePXE, err := cmd.Flags().GetBool("pxe")
 			if err != nil {
 				return err
 			}
-			return nodejoiner.NewAddNodesCommand(dir, kubeConfig)
+			return nodejoiner.NewAddNodesCommand(dir, kubeConfig, generatePXE)
 		},
 	}
+	nodesAddCmd.Flags().BoolP("pxe", "p", false, "Instead of an ISO, generates PXE files that can be used to boot the configured nodes to let them join an existing cluster")
```

Comment:
```suggestion
	nodesAddCmd.Flags().BoolP("pxe", "p", false, "Instead of an ISO, generates PXE artifacts that can be used to boot the configured nodes to let them join an existing cluster")
```